Makefile

```diff
@@ -1,6 +1,6 @@
 # Copyright (c) 2019-2020 Status Research & Development GmbH. Licensed under
 # either of:
-# - Apache License, version 2.0
+#sh- Apache License, version 2.0
```

Comment:
Huh?

Reply:
hm, don't remember doing that tbh, will remove